src/Config/Validator/Deploy/PhpVersion.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Config\Validator\Deploy;
+
+use Composer\Package\Version\VersionParser;
+use Composer\Semver\Constraint\ConstraintInterface;
+use Composer\Semver\Constraint\MultiConstraint;
+use Magento\MagentoCloud\Config\ValidatorInterface;
+use Magento\MagentoCloud\Config\Validator;
+use Magento\MagentoCloud\Package\MagentoVersion;
+use Magento\MagentoCloud\Package\Manager as PackageManager;
+use Magento\MagentoCloud\Config\Validator\Deploy\PhpVersion\ConstraintFactory;
+use Psr\Log\LoggerInterface;
+use Composer\Semver\Semver;
+
+/**
+ * Validates PHP version
```

Comment:
Please provide more detailed description. What rules of validation

Reply:
Done